packages/cli/src/lib/deploy/Deployer.ts

```diff
@@ -1,28 +1,32 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 
-import { Deployer } from "./deployer";
 import { loadDeployManifest, loadDeployManifestExt } from "../project";
 import { CacheDirectory } from "../CacheDirectory";
 import { Logger } from "../logging";
 
+import { Uri } from "@polywrap/core-js";
 import { DeployManifest } from "@polywrap/polywrap-manifest-types-js";
 import { Schema as JsonSchema } from "jsonschema";
 import path from "path";
 import nodePath from "path";
 
-interface PolywrapDeployConfig {
+export interface DeployModule {
```

Comment:
One nitpick - would it make sense to export this `DeployModule` into its own file?

It usually makes searching through the projects/files trivial, and it's the general approach I take when refactoring/coding new stuff.

Reply:
Agreed, going to push an update to make this the case within the `lib/deploy/` directory.